src/GRA.Domain.Service/ActivityService.cs

```diff
@@ -332,12 +332,13 @@ int pointsEarned
             }
         }
 
-        public async Task<int> AddBookAsync(int userId, Book book, bool addNotification = false)
+        public async Task<ServiceResult<int>> AddBookAsync(int userId, Book book, bool addNotification = false)
         {
             VerifyCanLog();
             int activeUserId = GetActiveUserId();
             var activeUser = await _userRepository.GetByIdAsync(activeUserId);
             int authUserId = GetClaimId(ClaimType.UserId);
+            ServiceResult<int> serviceResult = new ServiceResult<int>();
```

Comment:
We can use `var` here since we're making a `new ServiceResult` - it's clear what the type of the variable will be.